topostats/run_topostats.py

```diff
@@ -427,6 +430,7 @@ def main():
         process_scan,
         base_dir=config["base_dir"],
         filter_config=config["filter"],
+        scars_config=config["remove_scars"],
```

Comment:
Same as above...

If the `scars_config` section is going to remain as an independent top-level dictionary in `default_config.yaml` then this will be required.

But there is the option of moving the whole `remove_scars` configurations and nesting it under the `filter:` section.  If this is going to be done then you wouldn't need to pass `scars_config` in as a separate argument to `process_scan` as it is nested within `filter_config`.